package.json

```diff
@@ -31,7 +28,6 @@
     "passport-github2": "^0.1.12",
     "popper.js": "^1.16.1",
```

Comment:
I ran `npx depcheck` and found that `popper.js` is also unused now

Reply:
A heads up though, that tool also mentioned other `devDependencies` and `dependencies` that were unused but I found some uses for the other ones mentioned.

```
Unused dependencies
* @types/underscore
* popper.js
Unused devDependencies
* eslint-plugin-import
* husky
* webpack-cli
```

Reply:
Fixed.

Reply:
Thanks! One of those was unused, the others were used, but just not directly by name.